oasst-shared/oasst_shared/schemas/protocol.py

```diff
@@ -272,36 +272,25 @@ def __new__(cls, label: str, display_text: str = "", help_text: str = None):
         obj.help_text = help_text
         return obj
 
-    spam = "spam"
+    spam = "spam", "Seems to be intentionally low-quality or irrelevant"
     fails_task = "fails_task", "Fails to follow the correct instruction / task"
     not_appropriate = "not_appropriate", "Inappropriate for customer assistant"
     violence = "violence", "Encourages or fails to discourage violence/abuse/terrorism/self-harm"
     harmful = (
         "harmful",
         "Harmful content",
-        "The advice given in the output is harmful or counter-productive. This may be in addition to, but is distinct from the question about encouraging violence/abuse/terrorism/self-harm.",
+        "The advice given in the output is harmful or counter-productive. This may be in addition to, but is distinct from the label for encouraging violence/abuse/terrorism/self-harm.",
     )
     sexual_content = "sexual_content", "Contains sexual content"
-    toxicity = "toxicity"
+    toxicity = "toxicity", "Contains rude, abusive, profane or insulting content"
     moral_judgement = "moral_judgement", "Expresses moral judgement"
-    political_content = "political_content"
-    humor = "humor"
-    sarcasm = "sarcasm"
-    hate_speech = "hate_speech"
-    profanity = "profanity"
-    ad_hominem = "ad_hominem"
-    insult = "insult"
-    threat = "threat"
-    aggressive = "aggressive"
-    misleading = "misleading"
-    helpful = "helpful"
-    formal = "formal"
-    cringe = "cringe"
-    creative = "creative"
-    beautiful = "beautiful"
-    informative = "informative"
-    based = "based"
-    slang = "slang"
+    political_content = "political_content", "Expresses political views"
+    humor = "humor", "Contains humorous content including sarcasm"
+    hate_speech = "hate_speech", "Expresses sentiment which is discriminatory against a grouping of people"
```

Comment:
too general. in plain, "discriminate" just means making a difference, thus we discriminate against groups of people all the time justifiably.
Oxford has "abusive or threatening speech or writing that expresses prejudice on the basis of ethnicity, religion, sexual orientation, or similar grounds", I'd go with something like that or even more explicit